KsApi/MockService.swift

```diff
@@ -142,6 +139,11 @@ internal struct MockService: ServiceType {
 
   fileprivate let updateUserSelfError: ErrorEnvelope?
 
+  fileprivate let unwatchProjectMutationResult: Result<GraphMutationUnwatchProjectResponseEnvelope,
+  GraphError>?
+
+  fileprivate let watchProjectMutationResult: Result<GraphMutationWatchProjectResponseEnvelope, GraphError>?
```

Comment:
I'm planning to spend some time to change all of these to `Result<T, Error>` types.